### ISSUE TITLE
Ripple effect doesn't work

### ISSUE THREAD

**Issue by sadoroqw** [stickoking/razer-macos#63]
When I give razer macOS permission to accessibility for the ripple effect and when I select a colour, it just makes my keyboard black. (Blackwidow v4 pro)

**Comment by Ezekielman** [comment]
I have the same issue. It also keeps asking for permission after granting it.

![Image](https://github.com/user-attachments/assets/198765fc-b505-481a-8d8f-a8551811d405)
